package.json

```diff
@@ -12,8 +12,10 @@
     "start:update": "./scripts/update.sh",
     "debug": "npm run build -- --watch --debug",
     "licenses": "license-report --config=../.license-report-config.json > dist/licenses.html",
-    "lint": "eslint src/",
-    "test": "node --test && npm run lint",
+    "lint": "eslint",
+    "test": "npm run build && npm run wdio",
```

Comment:
not sure if `build` should be a part of the `test`. cannot we make user to build it by themselves before running tests?